tests/util/daemon.rs

```diff
@@ -10,19 +10,22 @@ use std::time::Duration;
 /// tests more reliable when they dispatch asynchronously to `dura serve`. However, nothing abot
 /// this is intended to be specific to dura.
 pub struct Daemon {
-   mailbox: Receiver<Option<String>>, 
-   pub child: Child,
-   /// Signals to kill daemon thread if this goes <= 0, like a CountDownLatch
-   kill_sign: Arc<Mutex<i32>>,
+    mailbox: Receiver<Option<String>>,
```

Comment:
i'm not sure why my editor started doing 3 space tabs. i'm embarrassed, actually. i saw it before but couldn't make sense of what was going on

Reply:
FWIW, I've betrayed my editor because of Rust 😄